README.md

```diff
@@ -1,2 +1,39 @@
-# books-with-es6
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Books list with ES6
+
+> Updatable books list coded with ES6 features
+
+
+## Built With
+
+- HTML, CSS and Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://sirvelasque.github.io/books-with-es6/)
```

Comment:
- [ ] Nice job with deploying the page 👍 Please, consider fixing links on the deployed version, which are not responding:
![2022-02-25 02 28 18](https://user-images.githubusercontent.com/49064106/155618110-2798cfe7-6b2c-4c0a-aa4e-538901cd1828.gif)

Reply:
hello! @elmar8287  and thanks!
I dont find a way to solve this problem 😞 it properly works on a local server but I dont find an answer that doesn't include react
![image](https://user-images.githubusercontent.com/82236725/155668262-bbf28052-720e-4e61-a3b2-b8070278983e.png)

Reply:
@Sirvelasque, do not worry, we will try to solve it together 👍 Could you first describe how you deploy your project? Considering your answer, I can support you correctly.

Reply:
@elmar8287 Thanks again
well, I went into the settings in the repository > pages
like this:
![image](https://user-images.githubusercontent.com/82236725/155670851-42cf9270-277d-4d7c-bf2d-e811185e86b2.png)

Reply:
@Sirvelasque, actually the branch is correctly selected. It should be investigated deeply. You can go on with all other comments fixing and ask for a review 👍